From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@animetauren__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@animetauren : Thanks for your contribution to the Azure documentation! The author, @yushwang, has been notified to review your proposed change.

yushwang commented:
Hi @animetauren, for VNet-to-VNet, either same subscription or different subscriptions, the EnableBGP must be set to True or $True on BOTH Connection resource objects. Whether or not it's $True or True on each connection should not matter. Please let me know if this is inaccurate.

@tysonn please close this PR.

animetauren commented:
@yushwang it is critical to have the $true, if you use just "true" and then run the script, yes the script will run and the vnet-to-vnet will be created but the connection status will stay at "unknown" and never succeed to "connected". This was seen in a real world application. To fix this we had to tear down the vnet-to-vnet connection and use "$true" and minutes later the connection became "connected".

yushwang commented:
#sign-off

Thanks!

MarkMcGeeAtAquent committed:
fix

MarkMcGeeAtAquent committed:
fix?

MarkMcGeeAtAquent committed:
last fix

dominicbetts committed:
Update screenshots

dominicbetts committed:
Acrolinx suggestions

cherylmc committed:
graphics

roygara committed:
Incorporating customer feedback.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rbacUpdate

cherylmc committed:
graphic

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into portalqs

roygara committed:
Fixing broken link.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rbacUpdate

roygara committed:
Robert feedback.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rbacUpdate

billmath committed:
part 2 of the pim release

JasonWHowell committed:
Updating broken forum links throughout Stream Analytics.

JasonWHowell committed:
Updating broken forum links throughout Stream Analytics

jpconnock committed:
Updating Manager metadata

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updateManager

JasonWHowell committed:
Updating broken forum links throughout Stream Analytics

tamram committed:
adding include to qs for getting account credentials

MohitGargMSFT committed:
(ForumsTeam) Fixes MicrosoftDocs/azure-docs#6713

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#6713

davidmu1 committed:
fix for git issue 6602

JasonWHowell committed:
Raising low acrolinx scores.

PRMerger12 committed:
Merge pull request #37068 from MohitGargMSFT/patch-10

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#6713

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into updateManager

billmath committed:
fixing links

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into portalqs

tamram committed:
fix links to images

JasonWHowell committed:
Undoing date changes

JasonWHowell committed:
Reverting minor changes

PRMerger18 committed:
Merge pull request #37061 from JasonWHowell/streamanalytics4

Updating broken forum links throughout Stream Analytics

kgremban committed:
Update tutorial section of IoT Edge

ktoliver committed:
Merge pull request #37065 from tamram/tamram-0404

adding include to qs for getting account credentials

Juliako committed:
Update media-services-portal-publish.md

GitHubber17 committed:
Merge pull request #37062 from JasonWHowell/streamanalytics5

Updating broken forum links throughout Stream Analytics

BryanTrach-MSFT committed:
fixes MicrosoftDocs/azure-docs#6717

Updated doc to use thumbnails instead of thumbs in three places. This ensures readers can easily move from step 1 to step 2.

gatneil committed:
made modify doc more clear

Juliako committed:
Update video-indexer-embed-widgets.md

PRMerger12 committed:
Merge pull request #37075 from Juliako/patch-3

Update media-services-portal-publish.md

kgremban committed:
Remove tutorial elements from edge how-to articles

hpsin committed:
Add optional claims link

PRMerger13 committed:
Merge pull request #37080 from BryanTrach-MSFT/patch-12

fixes MicrosoftDocs/azure-docs#6717

PRMerger13 committed:
Merge pull request #37060 from JasonWHowell/streamanalytics3

Updating broken forum links throughout Stream Analytics.

PRMerger14 committed:
Merge pull request #37081 from gatneil/modifyFeedback

made modify doc more clear

ktoliver committed:
Merge pull request #37049 from cherylmc/portalqs

graphics

PRMerger15 committed:
Merge pull request #37083 from hpsin/patch-5

Add optional claims link

kgremban committed:
Update TOC.yml

Commit:
Update monitoring CLI commands to CLI 2.0.

JasonWHowell committed:
Updating vs link

ktoliver committed:
Merge pull request #37076 from kgremban/apr4-edgetoc

Update tutorial section of IoT Edge

Juliako committed:
Update video-indexer-embed-widgets.md

Juliako committed:
Update video-indexer-embed-widgets.md

Commit:
Merge pull request #37088 from JasonWHowell/streamanalytics1

Updating VS link

jpconnock committed:
Updating Manager metadata

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Fix spacing issues with CLI commands.

neilpeterson committed:
[AKS] fixes microsoft/azure-docs#6674

jpconnock committed:
Manager metadata update:  VMs Windows [a-e]

PRMerger12 committed:
Merge pull request #37082 from Juliako/patch-5

Update video-indexer-embed-widgets.md

neilpeterson committed:
[AKS] note on upgrade paths

neilpeterson committed:
wording

v-mavick committed:
Correcting broken link to Free Azure subscription Web page.

Commit:
Merge pull request #37098 from v-mavick/door-monitor

Correcting broken link to Free Azure subscription Web page.

gokhanuluderya-msft committed:
Update experimentation-service-configuration-reference.md

PRMerger12 committed:
Merge pull request #37099 from gokhanuluderya-msft/patch-1

Update experimentation-service-configuration-reference.md

neilpeterson committed:
updates

Commit:
corrections to fix issue 6711

JasonWHowell committed:
Metadata update batch 1

Commit:
updates

JasonWHowell committed:
Metadata update batch 2

JasonWHowell committed:
Metadata batch 3

JasonWHowell committed:
Metadata update batch 4

JasonWHowell committed:
Metadata batch update 5

JasonWHowell committed:
Metadata update batch 6

JasonWHowell committed:
Metadata update only

JasonWHowell committed:
Merge branch 'master' into metadata1

JasonWHowell committed:
Update stream-analytics-documentdb-output.md

ajaycode committed:
added link to new pricing model in network-monitoring-overview

JasonWHowell committed:
Update stream-analytics-tools-for-visual-studio-edge-jobs.md

JasonWHowell committed:
Update stream-analytics-tools-for-visual-studio-install.md

JasonWHowell committed:
Update stream-analytics-tools-for-visual-studio-cicd.md

PRMerger18 committed:
Merge pull request #37097 from neilpeterson/aks-version-patch

[AKS] note on upgrade paths

ktoliver committed:
Merge pull request #37085 from kgremban/apr4-edgescrub

Remove tutorial elements from edge how-to articles

JasonWHowell committed:
Update stream-analytics-use-reference-data.md

ajaycode committed:
changed scheme to model

ajaycode committed:
added absolute URL instead of relative URL

PRMerger14 committed:
Merge pull request #37109 from ajaycode/network-monitoring-overview

added link to new pricing model in network-monitoring-overview

JasonWHowell committed:
Update backup-azure-files.md

JasonWHowell committed:
Update troubleshoot-azure-files.md

neilpeterson committed:
date correction

JasonWHowell committed:
Fixing rogue metadata for keyword tag

PRMerger20 committed:
Merge pull request #37110 from JasonWHowell/patch-1

Fixing broken bookmarks - Update backup-azure-files.md

PRMerger20 committed:
Merge pull request #37111 from JasonWHowell/patch-2

Fixing broken bookmarks - Update troubleshoot-azure-files.md

Commit:
Merge pull request #37112 from JasonWHowell/keywords

Fixing rogue metadata for keyword tag

ajaycode committed:
References section

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

dominicbetts committed:
Add metadata

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master'

dominicbetts committed:
Fix typo

codemillmatt committed:
Update article date

Commit:
Merge pull request #37115 from codemillmatt/xam-cosmos-sql-qs-date-change

Updating article date

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-cosmos-sql-qs-date-change

codemillmatt committed:
Update link address for quick start in experimental YML

PRMerger14 committed:
Merge pull request #37113 from ajaycode/network-monitoring-overview

References section

jaredr80 committed:
Add france south/france central under europe geo

Commit:
Merge pull request #37118 from jaredr80/patch-9

Add france south/france central under europe geo

Commit:
Added CLI and made edits.

Commit:
Added text about installation.

Commit:
Added sentence about installation.

Commit:
Update.

Commit:
Fix

PRMerger20 committed:
Merge pull request #37122 from jimdial/patch-129

Added sentence about installation.

Commit:
Merge pull request #37121 from jimdial/patch-128

Added text about installation.

amanarneja committed:
Fixes MicrosoftDocs/azure-docs#6342

Headings were not the consistent

PRMerger15 committed:
Merge pull request #37069 from davidmu1/gitissue6602

fix for git issue 6602

davidmu1 committed:
added cloud shell

Brenduns committed:
New post install issue for 1803 and 1802.

v-shils committed:
Merge pull request #36991 from dominicbetts/update-device-explorer

Update screenshots

Brenduns committed:
New post install issue for 1803 and 1802.

mimig1 committed:
Cosmos DB: Taking down experiment page

rmca14 committed:
Merge pull request #37119 from jimdial/patch-127

Added CLI and made edits.

davidmu1 committed:
added cloud shell

ktoliver committed:
Merge pull request #37101 from JasonWHowell/metadata7

Metadata update only

v-shils committed:
Merge pull request #37072 from jpconnock/updateManager

Update manager

rmca14 committed:
Merge pull request #37116 from codemillmatt/xam-cosmos-sql-qs-date-change

Link address typo fix

rmca14 committed:
Merge pull request #37091 from neilpeterson/acs-oms-patch

[AKS] fixes microsoft/azure-docs#6674

ktoliver committed:
Merge pull request #37102 from JasonWHowell/metadata5

Stream Analytics Metadata batch update 5

ktoliver committed:
Merge pull request #37103 from JasonWHowell/metadata6

Stream Analytics Metadata update batch 6

rmca14 committed:
Merge pull request #37087 from sptramer/monitoring/convert-cli-2.0

Update monitoring CLI commands to CLI 2.0.

ktoliver committed:
Merge pull request #37104 from JasonWHowell/metadata4

Stream Analytics Metadata update batch 4

ktoliver committed:
Merge pull request #37105 from JasonWHowell/metadata1

Stream Analytics Metadata update batch 1

rmca14 committed:
Merge pull request #37126 from davidmu1/gitissue6679

added cloud shell

ktoliver committed:
Merge pull request #37106 from JasonWHowell/metadata2

Stream Analytics Metadata update batch 2

rmca14 committed:
Merge pull request #37127 from Brenduns/update-1803-notes

New post install issue for 1803 and 1802.

PRMerger19 committed:
Merge pull request #37124 from amanarneja/patch-22

Fixes MicrosoftDocs/azure-docs#6342

ktoliver committed:
Merge pull request #37107 from JasonWHowell/metadata3

Stream Analytics Metadata batch 3

Commit:
Merge pull request #36950 from MarkMcGeeAtAquent/table-user-story-part2

Cosmos DB: fixed formatting for Jade snippet, fixed cmdlet name

Ja-Dunn committed:
Merge pull request #37092 from jpconnock/master

Update Manager metadata

Ja-Dunn committed:
Merge pull request #37100 from georgewallace/issue6711

automation : corrections to fix issue 6711

Ja-Dunn committed:
Merge pull request #37129 from mimig1/04052018

Cosmos DB: Taking down experiment page

v-alje committed:
Resolving merge conflict

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into pimApril

billmath committed:
adding Azure to title metadata

PRMerger15 committed:
Merge pull request #37050 from roygara/rbacUpdate

Updating managed disks docs with github issue feedback.

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

v-alje committed:
Merge pull request #37133 from MicrosoftDocs/FromPublicRepo

From public repo

itechedit committed:
Merge pull request #37058 from billmath/pimApril

part 2 of the pim release

PRMerger20 committed:
Merge pull request #37134 from CarlRabeler/patch-49

Update sql-database-long-term-backup-retention-configure.md

itechedit committed:
Merge pull request #37114 from JasonWHowell/patch-1

Update stream-analytics-build-an-iot-solution-using-stream-analytics.md fixes MicrosoftDocs/azure-docs#6549
